client/src/pages/Search/Search.tsx

```diff
@@ -2,30 +2,39 @@ import ResultCard from './ResultCard/ResultCard';
 import {} from '@/components/ui/drawer';
 import DrawerFilter from './Filter/DrawerFilter';
 import AsideFilter from './Filter/AsideFilter';
-import { useSearchParams } from 'react-router-dom';
-import { useEffect } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { useSearch } from '@/hooks/swr-hooks/useSearch';
+import { Button } from '@/components/ui/button';
 
 export default function Search() {
+  const [page, setPage] = useState(1);
   const [searchParams, _] = useSearchParams({});
-  useEffect(() => {
-    console.log(searchParams);
-  }, [searchParams]);
+  const { data, isLoading, error } = useSearch(
+    searchParams.get('query') || '',
+    page
+  );
+  console.log(data);
+
   return (
     <div className="max-w-[1280px] mx-auto min-h-[400px] sm:grid sm:grid-cols-10 sm:gap-x-4">
       <div className="flex flex-row justify-around sm:hidden">
         <DrawerFilter />
       </div>
       <AsideFilter />
-      <div className="p-4 sm:col-span-7">
-        {/* Only for testing */}
-        <div className="min-h-[50px] mb-4 p-4 border text-center">
-          <p className="font-bold">Showing search results for: </p>
-          <p>Query: {searchParams.get('query')}</p>
-          <p>Filter1: {searchParams.get('filter1')}</p>
-          <p>Filter2: {searchParams.get('filter2')}</p>
-          <p>Filter3: {searchParams.get('filter3')}</p>
-        </div>
-        <ResultCard />
+      <div className="p-4 flex flex-col gap-y-4 sm:col-span-7">
+        {data?.data.map((res) => (
+          <Link to={`/preview/${res._id}`} target="_blank" key={res._id}>
```

Comment:
- Declare the routes as a const in a separate constant file. If necessary, changing the names of one or more routes would be easy.
- I was expecting the route as '/filePreview' as declared in backend